content/api/metrics.apib

```diff
@@ -69,6 +69,149 @@ For example, requesting data between `4:22` to `6:37` would return results withi
 
 All precisions are valid up to a 24 hour time window. `15min` precision is valid up to a 48 hour time window. Hour precision is valid up to a 31 day time window.
 
+
+## Advanced Filters
+Each Metrics endpoint specifies certain filters that can be set as query parameters. In addition to these basic filter types, the Metrics API allows for more advanced filtering with multiple conditions, groupings, and comparators.
+
+The query `query_filters` query parameter must be set in order to use advanced filters. The value is presented as a URL-encoded JSON object that matches the schema detailed below.
+
+Note: this type of filtering replaces other filters specified as stand-alone query parameters. 
+e.g. in `?domains=mailing.com,query_filters={}`, the domains query parameter will be ignored. Other query parameters like `from`, `to`, `metrics`, `limit`, etc., will be processed as expected.
+
+The `query_filters` value is validated against a JSON schema. The schema can be retrieved through the `/deliverability/query-filters-schema` endpoint. See: [Advanced Query JSON Schema](#header-advanced-query-json-schema).
+
+### Groupings Structure 
+The JSON object consists of a top level "groupings" key that has an array of compare objects
+
+Example:
+````json
+{
+  "groupings": [
+    {
+       ... compare group 
+    },
+    {
+       ... compare group 
+    }
+  ]
+}
+
+````
+**Compare Group**
+
+The compare objects have a single logical comparison operator (`OR` or `AND`) that will be used to compare all of the filter conditions in the group
+
+Example:
+````json
+{
+      "AND":{
+        "domains":{
+          "eq":[
+            "gmail.com",
+            "yahoo.com",
+            "hotmail.com"
+          ],
+          "like":[
+            "mail"
+          ]
+        },
+        "sending_domains":{
+          "notEq": [ 
+            "sparkpost.com" 
+          ],
+          "notLike": [
+            ".io"
+          ]
+        }
+      }
+    }
+````
+
+Each filter group is comprised of the filter keys and comparator keywords. 
+
+**Filter Keys:**   
+               
+               domains
+               bindings
```

Comment:
Maybe we should put a link to Enterprise only or something to signify bindings / bound_groups are for enterprise and sending_ips / ip_pools are for SP customers